doc/faq/howto_faq.rst

```diff
@@ -423,18 +423,12 @@ locators as desired because the two axes are independent.
 Generate images without having a window appear
 ----------------------------------------------
 
-The easiest way to do this is use a non-interactive backend (see
-:ref:`what-is-a-backend`) such as Agg (for PNGs), PDF, SVG or PS.  In
-your figure-generating script, just call the
-:func:`matplotlib.use` directive before importing pylab or
-pyplot::
-
-    import matplotlib
-    matplotlib.use('Agg')
-    import matplotlib.pyplot as plt
-    plt.plot([1,2,3])
-    plt.savefig('myfig')
+Simply do not call `~matplotlib.pyplot.show`, and directly save the figure to
+the desired format::
 
+    import matplotlib.pyplot as plt
+    plt.plot([1, 2, 3])
+    plt.savefig('myfig.png')
```

Comment:
This is not a new problem, but this should probably include a note about closing the figure or you will eventually run out of memory.

Reply:
Actually perhaps the whole thing should just point to the same approach as the webserver solution (i.e. call the Figure() ctor) outside of pyplot.  Thoughts?